tests/org.jboss.tools.examples.ui.bot.test/README.md

```diff
@@ -3,18 +3,18 @@ Executing tests:
 mvn clean install -P{profile} -Dswtbot.test.skip=true -Dusage_reporting_enable=false -DdeployOnServer=true
 
 Where
-	${profile} select maven profile: WildFly11, EAP7, JavaEE7, SmokeTestsEAP7, SmokeTestsWildFly10, SmokeTestsJavaEE7 (WildFly10 profile is selected, if you don't specify profile)
+	${profile} select maven profile: WildFly14, EAP7, JavaEE7, SmokeTestsEAP7, SmokeTestsWildFly10, SmokeTestsJavaEE7 (WildFly10 profile is selected, if you don't specify profile)
 	
 You can specify other parameters as well(depending on profile you have selected):
 WildFly profile:
--PWildFly11
+-PWildFly14
 -DquickstartsURLWildFly=${quickstartsURLWildFly}		-	URL for WildFly Quickstarts
 -DexamplesFolderWildFly=${examplesFolderWildFly}		-	examples folder of WildFly Quickstarts
 
 G.e.
 
 ```
-mvn clean verify -DexamplesFolderWildFly=wildfly-11.0.0.Final-quickstarts -DquickstartsURLWildFly=https://github.com/wildfly/quickstart/releases/download/11.0.0.Final/wildfly-11.0.0.Final-quickstarts.zip -PWildFly -DspecificQuickstarts=helloworld-jms
+mvn clean verify -DexamplesFolderWildFly=wildfly-14.0.0.Final-quickstarts -DquickstartsURLWildFly=https://github.com/wildfly/quickstart/releases/download/14.0.0.Final/wildfly-14.0.0.Final-quickstarts.zip -PWildFly -DspecificQuickstarts=helloworld-jms
```

Comment:
in pom is profile WildFly14, can you please also change this line?

Reply:
Fixed in commit "JBIDE-26439 - updates README.md in org.jboss.tools.examples.ui.bot.test - changed maven code from -PWildFly to -PWildFly14 to match the POM profile."